packages/medusa/src/loaders/helpers/subscribers/__mocks__/index.ts

```diff
@@ -0,0 +1,18 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+
+export const eventBusServiceMock = {
+  subscribe: jest.fn().mockImplementation((...args) => {
+    return Promise.resolve(args)
+  }),
+}
+
+export const containerMock = {
```

Comment:
```suggestion
export const containerMock = createMedusaContainer().register({ [ModuleRegistrationName.EVENT_BUS: eventBusServiceMock })
```

Reply:
wouldnt that be enough?

Reply:
@adrien2p I didn't touch the existing implementation, and honestly I'd rather not in this PR as I just brought it back, and instead focus on getting the SDK work wrapped up 😄

Reply:
Ahah sorry, this is my good camper side, always improving and reducing debt around what i touch if we can. But no problem lets do that 🤘

Reply:
@adrien2p and I agree with you on that, it's just that it is quite a lot of code that i don't want to get into now, as I want to prioritize my main tasks. I think there are quite a few improvements to be done otherwise